config/samples/ofcir_v1_cipool.yaml

```diff
@@ -93,3 +93,32 @@ stringData:
       "endpoint": "https://172.22.0.3:6385",
       "image": "http://172.22.0.1/images/ofcir_image.qcow2"
     }
+---
+
+apiVersion: ofcir.openshift/v1
+kind: CIPool
+metadata:
+  name: cipool-equinix
+spec:
+  provider: equinix
+  priority: 0
+  size: 0
+  timeout: '4h'
+  state: available
+  type: host
+
+---
+
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cipool-equinix-secret
+type: Opaque
+stringData:
+  config: |
+    {
+      "projectid": "replace-with-project-id",
```

Comment:
What about the token? I was expecting it to be defined here

Reply:
https://github.com/packethost/packngo requires the token to be exported as env var PACKET_AUTH_TOKEN. Should I export it from the provider with os.Setenv?

Reply:
I think it could be possible also to use `packngo.NewClientWithAuth()` [function](https://github.com/packethost/packngo/blob/1d7ae3f7dbacd747ad8573c7daa246d0f8b2a9a8/packngo.go#L350-L353), which accepts an api token